docs/vault.md

```diff
@@ -81,6 +81,13 @@ vault-radar scan-confluence -outfile=confluence.csv -url="http://localhost:8090"
 An in-memory index keyed off of secret hashes will be generated prior to scanning the source. This index will be used to
 annotate whether a risk exists in Vault.
 
+### Generate an index file in offline mode
```

Comment:
The offline/online mode is also applicable to the `Compliance Reporting` mode, should we add a note about this above?